app/views/wishlist/index.html.haml

```diff
@@ -1 +1,14 @@
 %h1 Wishlist
+
+/ someone else should make this look pretty
+%table
+  %thead
+    %th Course
+    %th Actions
+  %tbody
+    - @wishlist.each do |item|
```

Comment:
Have you seen Rails collection rendering? http://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections